### PR TITLE
[11.80] Added new monster 'ink' race and fluid type

### PR DESCRIPTION
**Pull request by EPuncker**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

**Issues addressed:** 

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: src/const.h [hunk 1]
```diff
@@ -113,6 +113,7 @@ enum MagicEffectClasses : uint8_t {
 	CONST_ME_CRITICAL_DAMAGE = 173,
 	// 174 is empty
 	CONST_ME_PLUNGING_FISH = 175,
+	CONST_ME_DRAWINK = 202,
```
Comment (ranisalt) [inline src/const.h:116]:
```suggestion
	CONST_ME_DRAW_INK = 202,
```
?

File: src/game.cpp [hunk 2]
```diff
@@ -3913,6 +3913,15 @@ void Game::combatGetTypeInfo(CombatType_t combatType, Creature* target, TextColo
 					color = TEXTCOLOR_ELECTRICPURPLE;
 					effect = CONST_ME_ENERGYHIT;
 					break;
+				case RACE_INK:
+					color = TEXTCOLOR_BLACK;
+					effect = CONST_ME_DRAWINK;
```
Comment (ranisalt) [inline src/game.cpp:3918]:
```suggestion
					effect = CONST_ME_DRAW_INK;
```

File: src/const.h (hunk 1 above)
Comment (EPuncker) [reply src/const.h:116]: I just wanted to follow DRAWBLOOD pattern 😄

**Comment by omarcopires** [comment]
works fine

**Review by omarcopires — DISMISSED** [review]

**Review by DSpeichert — DISMISSED** [review]

**Review by DSpeichert — APPROVED** [review]

**Pull request merged**